modules/io/adaptors/read_oss_dataframe.cc

```diff
@@ -43,7 +43,7 @@ int main(int argc, const char** argv) {
   VINEYARD_CHECK_OK(client.Connect(ipc_socket));
   LOG(INFO) << "Connected to IPCServer: " << ipc_socket;
 
-  std::unique_ptr<OSSIOAdaptor> oss_io_adaptor(new OSSIOAdaptor(efile.c_str()));
+  auto oss_io_adaptor = IOFactory::CreateIOAdaptor(efile);
```

Comment:
`IOFactory` doesn't fit here, since the expection of "read_oss_dataframe" is reading oss, but if the parameter is a local file path, it doens't fail or even report proper errrors.